angular/projects/admin/src/app/services/user.service.ts

```diff
@@ -126,6 +128,20 @@ export class UserService {
       .collection<TanamUserInvited>('user-roles', queryFn).valueChanges();
   }
 
+  inviteUser(user: UserInvited) {
+    return this.siteCollection
+      .collection('user-roles').doc(user.email).set({
+        email: user.email,
+        role: user.role,
+        invited: firebase.firestore.FieldValue.serverTimestamp(),
+        updated: firebase.firestore.FieldValue.serverTimestamp()
+      });
+  }
+
+  removeInvitedUser(user: TanamUserInvited) {
```

Comment:
Use this one instead of the `UserInvited` above... and name it `UserInvitation` or `TanamUserInvitation`

Reply:
Resolved on this commit 
https://github.com/oddbit/tanam/pull/257/commits/0822192a47e16d5da4de432c77f9bc6ef519a0a3